### ISSUE TITLE
is it only for colors and no images? 

### ISSUE THREAD

**Issue by koloved** [LuFlo/texture_atlas_generator#1]
i took two mat with two images and it didnt packed into one texture ( its not work, its not texture packer, looks like color packer

**Comment by LuFlo** [comment]
Yes, it's only for single color materials, it uses the shader's color channel in order to build the map. If you want to use images you have to go through the baking process.

**Comment by Zenonair** [comment]
Its doesnt work for real textures. Its useless!

**Comment by LuFlo** [comment]
> Its doesnt work for real textures. Its useless!

Quite a few people are using it so it does not seem useless. I also state it clearly in the readme that this tool packs colors and creates a texture atlas out of the shader color channel.

Just use the built-in functionality of texture baking in blender if you want to use "real" textures. There are plenty of tutorials out there. If you want this functionality included to the plugin, feel free to open a PR :)
